deep-learning/README.md

```diff
@@ -0,0 +1,13 @@
+# Deep Learning Resources
+
+### Courses
+1. [Machine Learning by Andrew Ng](https://www.coursera.org/learn/machine-learning)
+2. [Deep Learning Specialization by Andrew Ng](https://www.coursera.org/specializations/deep-learning)
+3. [Machine Learning - Stanford CS229](http://cs229.stanford.edu/)
+3. [Convolutional Neural Networks for Visual Recognition - Stanford CS231n](http://cs231n.stanford.edu/)
```

Comment:
I think we can add names of student who have done these courses, so that if anyone want review can contact that man

Reply:
We can also define difficulty level and pre requisites for each courses

Reply:
We can include that in the end but not beside the links.

Reply:
we can also make a table @SharathRaparthy with headings as Course, difficulty, prerequisites, course completed by (his name with hyperlinked Linkedin profile)

Reply:
Table is not visually pleasing imo @take2rohit

Reply:
They can just ask on Discord for the review. This is just a list of resources (I would say no names 😬).